### DISCUSSION TITLE
怎么删除我的alist3账户从新开始

### DISCUSSION THREAD

**Discussion by seamountzh** [AlistGo/alist#3357] [Q&A]
我在存储里挂在123网盘的时候，不小心将挂在路径写为“”\123网盘“”，然后在管理页面无法显示已经挂载网盘，一直报错“No such column: index”，在主页下“123网盘”盘符也无法删除，切换过去就一直报错。不知道怎么处理好了，请教各位大神帮忙解决一下。万分感谢！！！
![image](https://user-images.githubusercontent.com/43403671/218262534-fb22dd27-368a-4f4d-af94-f628b5e09eff.png)

**Comment by 12189108** [comment]
删除data文件夹，重启alist就行
